salt/metalk8s/addons/dex/deployed/chart.sls

```diff
@@ -188,7 +212,7 @@ spec:
   template:
     metadata:
       annotations:
-        checksum/config: 278f2b27e9441887e4070365aa3df2ccf668da0025ebf565605555c2b7b16042
+        checksum/config: 82272216f0373dd6510af9cea925a8642900f1fe7400d2cbcf4ef54e59ac7d9c
```

Comment:
We're changing this when needed, right?

Reply:
I thought this checksum hashes are generated each time we render the charts? Don't we need to use the new values generated?
Also, when are we required to change it ?